### ISSUE TITLE
Subscriptions

### ISSUE THREAD

**Issue by bennypowers** [reach-digital/polymer-apollo-client#10]
There should be something like

```html
<graphql-subscription result="{{post}}">
  <script type="application/graphql">
    subscription post(postId: ID!) {
      id
      title
      photo
      content
      totalLikes
    }
  </script>
</graphql-subscription>

<paper-card heading="[[post.post.title]]" image="[[post.post.image]]">
  <div class="card-content">[[post.post.content]]</div>
  <div class="card-actions">
    [[post.post.totalLikes]] Likes!
    <paper-icon-button icon="like" on-click="like" data-post-id$="[[post.post.id]]"></paper-icon-button>
  </div>
</paper-card>
```

The best would be if we can somehow use optimistic ui to set paper-icon-button's active status, so I can indicate with css that I've liked the post

```css
paper-icon-button[active] {
  color: pink;
}
```

**Comment by FluorescentHallucinogen** [comment]
+1

**Comment by FluorescentHallucinogen** [comment]
Good news! I've implemented an early proof-of-concept. Will create pull request soon. 😉

**Comment by FluorescentHallucinogen** [comment]
See #15. :wink:

**Comment by FluorescentHallucinogen** [comment]
@paales Please close this issue. It fixed by #15.
